### DISCUSSION TITLE
OctoDash with no internet connection!

### DISCUSSION THREAD

**Discussion by robbjosh** [UnchartedBull/OctoDash#1255] [General]
Hi guys,

I need some assistance. I have OctoPi setup and running. I have a 5" Waveshare touchscreen connected and working. Everything is work, but when I disconnect my Ethernet cable I get errors on OctoDash and the only to get them to disappear is to reconnect the internet/Ethernet cable. I want to use OctoPi with OctoDash as my printer has no display, however it seems I need to have an internet connection to use it. The error is something about cant retrieve layer progress http failure responce.

Is there any way I can use OctoDash without an internet connection?

**Comment by UnchartedBull** [comment]
Change your host in your settings to localhost / 127.0.0.1. Currently it probably tries to access something like 192.x.x.x or octopi.local, which all require the interface to be up. You can also manually set the interface to up, but that is a little more effort.

**Reply by DLXXV-Kelvin** [reply]
This doesn't work for me. In fact, it won't finish loading at all now. It only works if I have wifi on and the original setting of I think it was octopi.local:80. Any ideas?

**Reply by UnchartedBull** [reply]
localhost should always be available. Did you manually edit your internet interfaces maybe?

**Comment by robbjosh** [comment]
Thank you kind Sir. This has fixed the problem

**Comment by Mech131** [comment]
I'm also having this problem. It is set to octopi.local:80 and it only works when connected to the network. If the network is offline octodash stops working.

**Reply by UnchartedBull** [reply]
Yes that's expected since octopi.local needs to be resolved, please use localhost and everything should work.

**Reply by Mech131** [reply]
Thats not working. I tried http://localhost, localhost, 127.0.0.1, http://127.0.0.1 and i keep getting the 'socket is dead' error message.

Edit: Had to put the port number after it. http://127.0.0.1:80/ works
